`uvicorn` fails with a weird error in the test environment on `win-64`:
```
gw2] [ 69%] PASSED tests/test_config.py::test_log_config_json 
[gw0] [ 69%] PASSED tests/supervisors/test_reload.py::test_base_reloader_should_exit 
tests/test_config.py::test_log_config_yaml[log_config.yml] 
tests/supervisors/test_reload.py::test_base_reloader_closes_sockets_on_shutdown 
Leaving build/test directories:
  Work:
 C:\Users\task_176821365972709\croot\work 
  Test:
 C:\Users\task_176821365972709\croot\test_tmp 
Leaving build/test environments:
  Test:
source activate  C:\Users\task_176821365972709\croot\_test_env 
  Build:
source activate  C:\Users\task_176821365972709\croot\_build_env 


Failed recipes:
^CTerminate batch job (Y/N)? 
Traceback (most recent call last):
  File "C:\miniconda3\Lib\site-packages\conda\exception_handler.py", line 29, in __call__
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\miniconda3\Lib\site-packages\conda\cli\main.py", line 61, in main_subshell
    exit_code = do_call(args, parser)
                ^^^^^^^^^^^^^^^^^^^^^
  File "C:\miniconda3\Lib\site-packages\conda\cli\conda_argparse.py", line 205, in do_call
    result = getattr(module, func_name)(args, parser)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\miniconda3\Lib\site-packages\conda\cli\main_run.py", line 109, in execute
    response = subprocess_call(
               ^^^^^^^^^^^^^^^^
  File "C:\miniconda3\Lib\site-packages\conda\gateways\subprocess.py", line 121, in subprocess_call
    stdout, stderr = process.communicate(input=stdin)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\miniconda3\Lib\subprocess.py", line 1201, in communicate
    self.wait()
  File "C:\miniconda3\Lib\subprocess.py", line 1264, in wait
    return self._wait(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\miniconda3\Lib\subprocess.py", line 1590, in _wait
    result = _winapi.WaitForSingleObject(self._handle,
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
conda.CondaError: KeyboardInterrupt

Terminate batch job (Y/N)? 
Traceback (most recent call last):
  File "C:\Users\task_176821365972709\test.py", line 239, in <module>
    sys.exit(main())
             ^^^^^^
  File "C:\Users\task_176821365972709\test.py", line 231, in main
    conda_test(dest_path, src_channels, conda_env)
  File "C:\Users\task_176821365972709\test.py", line 138, in conda_test
    result = subprocess.run(test_cmd, env=env, shell=use_shell)  # nosec
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\miniconda3\Lib\subprocess.py", line 550, in run
    stdout, stderr = process.communicate(input, timeout=timeout)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\miniconda3\Lib\subprocess.py", line 1201, in communicate
    self.wait()
  File "C:\miniconda3\Lib\subprocess.py", line 1264, in wait
    return self._wait(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\miniconda3\Lib\subprocess.py", line 1590, in _wait
    result = _winapi.WaitForSingleObject(self._handle,
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt
^CTerminate batch job (Y/N)? 

!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! KeyboardInterrupt !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
C:\Users\task_176821365972709\croot\uvicorn-standard_1768213996217\_test_env\lib\threading.py:324: KeyboardInterrupt
(to show a full traceback on KeyboardInterrupt use --full-trace)
Traceback (most recent call last):
  File "C:\Users\task_176821365972709\croot\uvicorn-standard_1768213996217\_test_env\Scripts\pytest-script.py", line 10, in <module>
    sys.exit(console_main())
  File "C:\Users\task_176821365972709\croot\uvicorn-standard_1768213996217\_test_env\lib\site-packages\_pytest\config\__init__.py", line 201, in console_main
    code = main()
  File "C:\Users\task_176821365972709\croot\uvicorn-standard_1768213996217\_test_env\lib\site-packages\_pytest\config\__init__.py", line 175, in main
    ret: ExitCode | int = config.hook.pytest_cmdline_main(config=config)
  File "C:\Users\task_176821365972709\croot\uvicorn-standard_1768213996217\_test_env\lib\site-packages\pluggy\_hooks.py", line 513, in __call__
    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)
  File "C:\Users\task_176821365972709\croot\uvicorn-standard_1768213996217\_test_env\lib\site-packages\pluggy\_manager.py", line 120, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File "C:\Users\task_176821365972709\croot\uvicorn-standard_1768213996217\_test_env\lib\site-packages\pluggy\_callers.py", line 139, in _multicall
    raise exception.with_traceback(exception.__traceback__)
  File "C:\Users\task_176821365972709\croot\uvicorn-standard_1768213996217\_test_env\lib\site-packages\pluggy\_callers.py", line 103, in _multicall
    res = hook_impl.function(*args)
  File "C:\Users\task_176821365972709\croot\uvicorn-standard_1768213996217\_test_env\lib\site-packages\_pytest\main.py", line 336, in pytest_cmdline_main
    return wrap_session(config, _main)
  File "C:\Users\task_176821365972709\croot\uvicorn-standard_1768213996217\_test_env\lib\site-packages\_pytest\main.py", line 324, in wrap_session
    config.hook.pytest_sessionfinish(
  File "C:\Users\task_176821365972709\croot\uvicorn-standard_1768213996217\_test_env\lib\site-packages\pluggy\_hooks.py", line 513, in __call__
    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)
  File "C:\Users\task_176821365972709\croot\uvicorn-standard_1768213996217\_test_env\lib\site-packages\pluggy\_manager.py", line 120, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File "C:\Users\task_176821365972709\croot\uvicorn-standard_1768213996217\_test_env\lib\site-packages\pluggy\_callers.py", line 139, in _multicall
    raise exception.with_traceback(exception.__traceback__)
  File "C:\Users\task_176821365972709\croot\uvicorn-standard_1768213996217\_test_env\lib\site-packages\pluggy\_callers.py", line 122, in _multicall
    teardown.throw(exception)  # type: ignore[union-attr]
  File "C:\Users\task_176821365972709\croot\uvicorn-standard_1768213996217\_test_env\lib\site-packages\_pytest\logging.py", line 873, in pytest_sessionfinish
    return (yield)
  File "C:\Users\task_176821365972709\croot\uvicorn-standard_1768213996217\_test_env\lib\site-packages\pluggy\_callers.py", line 122, in _multicall
    teardown.throw(exception)  # type: ignore[union-attr]
  File "C:\Users\task_176821365972709\croot\uvicorn-standard_1768213996217\_test_env\lib\site-packages\_pytest\terminal.py", line 936, in pytest_sessionfinish
    result = yield
  File "C:\Users\task_176821365972709\croot\uvicorn-standard_1768213996217\_test_env\lib\site-packages\pluggy\_callers.py", line 122, in _multicall
    teardown.throw(exception)  # type: ignore[union-attr]
  File "C:\Users\task_176821365972709\croot\uvicorn-standard_1768213996217\_test_env\lib\site-packages\_pytest\warnings.py", line 119, in pytest_sessionfinish
    return (yield)
  File "C:\Users\task_176821365972709\croot\uvicorn-standard_1768213996217\_test_env\lib\site-packages\pluggy\_callers.py", line 103, in _multicall
    res = hook_impl.function(*args)
  File "C:\Users\task_176821365972709\croot\uvicorn-standard_1768213996217\_test_env\lib\site-packages\xdist\dsession.py", line 99, in pytest_sessionfinish
    nm.teardown_nodes()
  File "C:\Users\task_176821365972709\croot\uvicorn-standard_1768213996217\_test_env\lib\site-packages\xdist\workermanage.py", line 117, in teardown_nodes
    self.group.terminate(self.EXIT_TIMEOUT)
  File "C:\Users\task_176821365972709\croot\uvicorn-standard_1768213996217\_test_env\lib\site-packages\execnet\multi.py", line 237, in terminate
    safe_terminate(
  File "C:\Users\task_176821365972709\croot\uvicorn-standard_1768213996217\_test_env\lib\site-packages\execnet\multi.py", line 348, in safe_terminate
    reply.get()
  File "C:\Users\task_176821365972709\croot\uvicorn-standard_1768213996217\_test_env\lib\site-packages\execnet\gateway_base.py", line 327, in get
    self.waitfinish(timeout)
  File "C:\Users\task_176821365972709\croot\uvicorn-standard_1768213996217\_test_env\lib\site-packages\execnet\gateway_base.py", line 334, in waitfinish
    if not self._result_ready.wait(timeout):
  File "C:\Users\task_176821365972709\croot\uvicorn-standard_1768213996217\_test_env\lib\threading.py", line 607, in wait
    signaled = self._cond.wait(timeout)
  File "C:\Users\task_176821365972709\croot\uvicorn-standard_1768213996217\_test_env\lib\threading.py", line 320, in wait
    waiter.acquire()
KeyboardInterrupt
[taskcluster 2026-01-12T10:34:00.830Z]                        Exit Code: 3221225786
[taskcluster 2026-01-12T10:34:00.830Z]                        User Time: 0s
[taskcluster 2026-01-12T10:34:00.830Z]                      Kernel Time: 31.25ms
[taskcluster 2026-01-12T10:34:00.830Z]                        Wall Time: 1m17.2437603s
[taskcluster 2026-01-12T10:34:00.830Z]  Average Available System Memory: 60.39 GiB
[taskcluster 2026-01-12T10:34:00.830Z]       Average System Memory Used: 2.90 GiB
[taskcluster 2026-01-12T10:34:00.830Z]          Peak System Memory Used: 3.56 GiB
[taskcluster 2026-01-12T10:34:00.830Z]              Total System Memory: 63.30 GiB
[taskcluster 2026-01-12T10:34:00.830Z]                           Result: FAILED
[taskcluster 2026-01-12T10:34:00.830Z] === Task Finished ===
[taskcluster 2026-01-12T10:34:00.830Z] Task Duration: 1m20.9123186s
[taskcluster 2026-01-12T10:34:00.886Z] Uploading link artifact public/logs/live.log to artifact public/logs/live_backing.log with expiry 2027-01-15T10:29:16.436Z
[taskcluster:error] exit status 0xc000013a
```

I skipped flaky tests on Windows

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -87,6 +87,7 @@ outputs:
         - websockets >=10.4
 
     {% set ignore_tests = '' %}
+    {% set tests_to_skip = '' %}
     # >       from websockets.server import ServerProtocol
     # E       ImportError: cannot import name 'ServerProtocol' from 'websockets.server'
     # https://github.com/python-websockets/websockets/blob/13.1/src/websockets/server.py#L48
@@ -100,7 +101,7 @@ outputs:
     {% set ignore_tests = ignore_tests + ' --ignore="tests/supervisors/test_multiprocess.py"' %}  # [not linux]
     {% set ignore_tests = ignore_tests + ' --ignore="tests/supervisors/test_reload.py"' %}  # [not linux]
 
-    {% set tests_to_skip = " test_http2_upgrade_request " %}
+    {% set tests_to_skip = tests_to_skip + " test_http2_upgrade_request " %}
     {% set tests_to_skip = tests_to_skip + " or test_proxy_headers_websocket_x_forwarded_proto " %}
     # Needs a2wsgi >=1.10.8
     {% set ignore_tests = ignore_tests + ' --ignore="tests/middleware/test_wsgi.py"' %}

